tests/Type/Doctrine/Query/QueryResultTypeWalkerTest.php

```diff
@@ -1310,6 +1312,8 @@ public function getTestData(): iterable
 			',
 		];
 
+		// There is no error after the merge of https://github.com/doctrine/dbal/pull/5755
+		$moduloError = strpos((new SqlitePlatform())->getModExpression('', ''), '%') === false;
```

Comment:
Dunno the full reason but the modExpression implementation was changed from 
```
Mod(A, B)
```
to
```
A % B
```
and this is not throwing an error anymore.